examples/01.Basics/Vibrato/Vibrato.ino

```diff
@@ -41,7 +45,8 @@ void updateControl(){
 
 
 AudioOutput_t updateAudio(){
-    Q15n16 vibrato = (Q15n16) intensity * aVibrato.next();
+  //Q15n16 vibrato = (Q15n16) intensity * aVibrato.next();
+  auto vibrato = intensity * SFixMath<0, 7>::fromRaw(aVibrato.next()); // Oscils in Mozzi are 8bits: 7bits of signal plus one bit of sign, so what they fit into a SFixMath<0,7> which is a signed fixMath type with 7 bits of value. 
```

Comment:
Use toSFraction(), or at least mention it as an alternative.

Reply:
Yes, I did not do the examples, planning for maybe a consecutive PR, but as these two examples are in this one this should be coherent.